xml/System.Data.SqlTypes/SqlByte.xml

```diff
@@ -1191,7 +1191,7 @@
         <summary>Performs a bitwise exclusive-OR operation on the supplied parameters.</summary>
         <returns>The results of the bitwise XOR operation.</returns>
         <remarks>
-          <format type="text/markdown"><![CDATA[The equivalent method for this operator is <xref:System.Data.SqlTypes.SqlByte.Xor%2A> System.Data.SqlTypes.SqlByte)?qualifyHint=True&autoUpgrade=False]]></format>
+          <format type="text/markdown"><![CDATA[The equivalent method for this operator is <xref:System.Data.SqlTypes.SqlByte.Xor(System.Data.SqlTypes.SqlByte,System.Data.SqlTypes.SqlByte)]]></format>
```

Comment:
broken link. it's missing the closing > for the xref link
and sentence is missing a period